libcloudforensics/providers/aws/forensics.py

```diff
@@ -164,7 +164,7 @@ def StartAnalysisVm(
     vm_name: str,
     default_availability_zone: str,
     boot_volume_size: int,
-    ami: str = '',
+    ami: Optional[str] = None,
```

Comment:
Doesn't `Optional[str]` default to `None`? e.g. do you need to specify `= None` each time? https://mypy.readthedocs.io/en/stable/kinds_of_types.html#optional-types-and-the-none-type

Reply:
If you remove `= None` it seems that you are changing the argument to a non-default parameter and my IDE is not happy.